recipes-browser/wpewebkit/wpewebkit_2.40.0.bb

```diff
@@ -0,0 +1,39 @@
+require wpewebkit.inc
+require conf/include/devupstream.inc
```

Comment:
you need to remove `require conf/include/devupstream.inc` from `recipes-browser/wpewebkit/wpewebkit_2.38.5.bb`